tests/src/test/java/org/eclipse/hono/tests/registry/TenantHttpIT.java

```diff
@@ -268,8 +300,26 @@ public void testGetAddedTenant(final VertxTestContext context)  {
     @Test
     public void testAddTenantFailsForMalformedTrustConfiguration(final VertxTestContext context) {
 
-        final Tenant requestBody = Tenants.createTenantForTrustAnchor("test-dn", "NotBased64Encoded".getBytes(), "RSA");
+        final Tenant requestBody = Tenants.createTenantForTrustAnchor("CN=ca, OU=Hono, O=Eclipse", "NotBased64Encoded".getBytes(), "RSA");
+
+        registry.addTenant(tenantId, requestBody, HttpURLConnection.HTTP_BAD_REQUEST).setHandler(context.completing());
+    }
 
+    /**
+     * Verifies that the service returns a 400 status code for an add tenant request with a <em>trusted-ca</em> configuration
+     * containing a JSON object as a value.
+     * 
+     * @param context The Vert.x test context.
+     */
+    @Test
+    public void testAddTenantFailsWhenOldTrustModelIsUsed(final VertxTestContext context) {
+        final JsonObject trustConfig = new JsonObject()
```

Comment:
I assume the test is for the registry to check if the trusted CA spec is an array of objects and not just an object. If so, then we can simply use an empty JsonObject here, can't we?

Reply:
sure 👍